mimic/model/cloudfeeds.py

```diff
@@ -0,0 +1,86 @@
+"""
+Emulate the behavior of Rackspace Cloud Feeds service.
+"""
+
+import attr
+
+
+@attr.s
+class CloudFeedsProduct(object):
+    """
+    Models a single CloudFeed product endpoint and its respective
+    functionality.
+    """
+    title = attr.ib(attr.validators.instance_of(str))
```

Comment:
This may not be as compatible with Python3 - we've been using `six.string_types` for our validation so it accepts both bytes and unicode:

```
from six import string_types
...
title = attr.ib(attr.validators.instance_of(string_types))
```